drizzlepac/haputils/svm_quality_analysis.py

```diff
@@ -830,28 +830,25 @@ def report_wcs(total_product_list, json_timestamp=None, json_time_since_epoch=No
     """
     log.setLevel(log_level)
     log.info('\n\n*****     Begin Quality Analysis Test: report_wcs.     *****\n')
+    apriori_suffix_list = ['HSC30', 'GSC240']
```

Comment:
This will work for now.  Unfortunately, if additional apriori solutions are defined, this code would need to be updated to correctly recognize them, which would probably not happen until problems start showing up (in the archive again?).  The absence of any of the aposteriori terms in the WCS and presence of the '-' in the WCSNAME would be a sufficient universal check of whether a WCSNAME is apriori.  This would work due to the fact that the aposteriori terms are fixed and will not change, ever, as far as I can imagine.

Reply:
Fix found in next commit.  Added comment for examples of WCS naming convention for default, a priori, and a posteriori solutions.